packages/eslint-plugin-svelte/src/rules/derived-has-same-inputs-outputs.ts

```diff
@@ -53,7 +55,20 @@ export default createRule('derived-has-same-inputs-outputs', {
 					node: fn,
 					loc: fnParam.loc,
 					messageId: 'unexpected',
-					data: { name: expectedName }
+					data: { name: expectedName },
+					fix: (fixer) => {
+						const scope = getSourceCode(context).getScope(fn.body);
+						const variable = scope.variables.find((variable) => variable.name === fnParam.name);
+
+						if (!variable) {
+							return fixer.replaceText(fnParam, expectedName);
+						}
+
+						return [
+							fixer.replaceText(fnParam, expectedName),
+							...variable.references.map((ref) => fixer.replaceText(ref.identifier, expectedName))
+						];
+					}
```

Comment:
I think it’s not possible to add a fixer because we need to check if `expectedName` is already used.
If `expectedName` is already used, I don’t think we can know how to rename it properly.

Reply:
we can check if the name already exists and only add a fixer if it does